server/routes/nl/api.py

```diff
@@ -42,9 +42,9 @@ def search_vector():
   """Performs vector search for a given query and embedding index.
   """
   query = request.args.get('query')
-  index = request.args.get('index')
+  idx = request.args.get('idx')
   if not query:
     flask.abort(400, 'Must provde a `query`')
-  if not index:
-    flask.abort(400, 'Must provde an `index`')
-  return dc.nl_search_vars([query], index)
+  if not idx:
+    flask.abort(400, 'Must provde an `idx`')
+  return dc.nl_search_vars([query], idx.split(','))
```

Comment:
nit: I wonder if instead of using ',' as a delimiter, could we do args.getList('idx') & pass the idx as a list argument

Reply:
In this case i think it's doable, but as we use "," delim in the other places, might just use the same convention here.